pysat/utils/files.py

```diff
@@ -203,26 +227,42 @@ def parse_fixed_width_filenames(files, format_str):
 def parse_delimited_filenames(files, format_str, delimiter):
     """Parse list of files, extracting data identified by format_str.
 
+    Will parse file using `delimiter` though the function does not require
+    every delimited item to be a variable, and more than one variable
+    may be within a delimited section. Thus, the main practical
+    difference with `parse_fixed_width_filenames` is more support for
+    the use of the wildcard '*' within `format_str`.
```

Comment:
Do we need the less adaptable routine, then?

Reply:
So far there have always been filenames out there with unanticipated challenges. Possible there are some still out there.

Reply:
The fixed width parser is also less prone to false positives.

Reply:
"The fixed width parser is also less prone to false positives."  Something like this should be added to the docstring.  It would also be good to adapt the instrument template to use this one and say in a block comment "switch to the other routine for these specific cases."

Reply:
I added that note but then changed a function setting that reduced the potential for false positives. I changed the note  to remind the user that using the '*' too much can lead to false positives. I also updated the template instrument comment and parsing method.